docs/product/base/transitions.html

```diff
@@ -32,8 +32,7 @@
     <p class="stacks-copy">The base transition applies a moderate duration, timing function, and delay to an element.</p>
     <div class="stacks-preview">
 {% highlight html linenos %}
-<div class="t">…</div>
-{% endhighlight %}
+<div class="t">…</div>{% endhighlight %}
```

Comment:
We actually make the highlighted code flush left with the left side of the document, since it's all a `<pre>` tag. It messes with the format of the doc itself, but the output is correct :v:

Reply:
Oh wow, I did not know this! Thanks for looking out 👍